.travis.yml

```
@@ -1,5 +1,5 @@
 language: node_js
-node_js: '11.14.0'
+node_js: 'node'
```

```suggestion
node_js: '12'
```

Se we don't have this all over again when 13 is released